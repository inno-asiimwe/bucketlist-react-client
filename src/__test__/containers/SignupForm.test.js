import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import RegisterUser from '../../containers/SignupForm';

describe('RegisterUser', () => {
  it('Matches its snapshot', () => {
    const output = shallow(<RegisterUser />)
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
