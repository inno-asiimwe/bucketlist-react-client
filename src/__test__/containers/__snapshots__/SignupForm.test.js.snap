// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterUser Matches its snapshot 1`] = `
<Connect(Form(Connect(RegisterUser)))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="RegisterForm"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
  validate={[Function]}
/>
`;
